File: Ad-Hoc/pagina33/3455/3455.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("\n")
const [grafica, dinamica, geometrica] = input.shift().split(" ").map(Number)
const questao_desejada = input.shift();

let total = 0;

if(questao_desejada == "A"){
    total += grafica
    total += dinamica * (3/2)
    total += geometrica * 2.5
}
else if (questao_desejada == "B"){
    total += dinamica
    total += grafica * (2/3)
    total += geometrica * 2.5 * (2/3)
}
else if (questao_desejada == "C"){
    total += geometrica
    total += grafica * (1 / 2.5)
    total += dinamica * (3/2) * (1 / 2.5)
}

console.log(Math.floor(total))

File: Ad-Hoc/__3000__/3455/3455.py
grafica, dinamica, geometria = [int(x) for x in input().split()]
tipo_questao_desejada = input().strip()

total = 0

if tipo_questao_desejada == "A":  # Gráfica
    total += grafica
    total += dinamica * (3 / 2)  # 1 dinâmica = 1.5 gráficas
    total += geometria * 2.5     # 1 geometria = 2.5 gráficas
elif tipo_questao_desejada == "B":  # Dinâmica
    total += dinamica
    total += grafica * (2 / 3)   # 1 gráfica = 0.666 dinâmica
    total += geometria * 2.5 * (2 / 3)  # geometria -> gráfica -> dinâmica
elif tipo_questao_desejada == "C":  # Geometria
    total += geometria
    total += grafica * (1 / 2.5)       # 1 gráfica = 0.4 geometria
    total += dinamica * (3 / 2) * (1 / 2.5)  # dinâmica -> gráfica -> geometria

print(int(total))